CHANGELOG.md

```diff
@@ -1,15 +1,29 @@
 ## UNRELEASED
 
+BREAKING CHANGES:
+* The consul-k8s repository has been merged with consul-helm and now contains the `consul-k8s-control-plane` binary (previously named `consul-k8s`) and the Helm chart to deploy Consul on Kubernetes. The docker image previously named `hashicorp/consul-k8s` has been renamed to `hashicorp/consul-k8s-control-plane`. The binary and Helm chart will be released together with the same version. [[GH-589](https://github.com/hashicorp/consul-k8s/pull/589)]
```

Comment:
Could we say something here about "If you're installing Consul through the Helm chart and you're not editing the `global.imageK8S` setting then you will have no issues"

Reply:
Yup!